src/routes/index.py

```diff
@@ -24,7 +26,17 @@ def get_blueprint():
 
 @index_blueprint.route("/")
 def index():
+    # If user is authenticated, get user recommendations based on saved ingredients and recipes
     if current_user.is_authenticated:
-        return render_template("index2.html", userdata=current_user.to_json())
+        data = (
+            "TESTING AUTH SECTION" + current_user.name,
+            current_user.email,
+            current_user.profile_pic,
+        )
+        # return render_template("index2.html", userdata=current_user.to_json())
+    # Else, get dummy data/random recommendations
     else:
-        return render_template("index.html")
+        data = get_recommended_recipes()
+        print(data)
```

Comment:
```suggestion
```

Reply:
These print statement is for testing how data will be displayed